content/pt-br/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -0,0 +1,478 @@
+---
+update_date: 2022-07-18
+origin_version: 1.24
+contributors: DonatoHorn
+reviewers: 
+- erictune
+- mikedanese
```

Comment:
```suggestion
```

Reply:
@DonatoHorn this section still needs to be removed, since the listed reviewers do not review content in Brazilian Portuguese.